amlb/resources.py

```diff
@@ -62,6 +63,48 @@ def project_info(self):
             branch=branch
         )
 
+    @lazy_property
+    def git_info(self):
+        def git(cmd, defval=None):
+            try:
+                return run_cmd(f"git {cmd}", _log_level_=logging.DEBUG)[0].strip()
+            except Exception:
+                return defval
+
+        na = "NA"
+        version = git("--version")
+        is_git_repo = version and git("rev-parse --git-dir 2> /dev/null")
+        if version and is_git_repo:
```

Comment:
Duplication since `if version and is_git_repo` expands to `if version and version and git("rev-parse --git-dir 2> /dev/null")`.
```suggestion
        if is_git_repo:
```